contracts/rewards/src/error.rs

```diff
@@ -0,0 +1,9 @@
+use axelar_wasm_std_derive::IntoContractError;
+use cosmwasm_std::StdError;
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq, IntoContractError)]
+pub enum ContractError {
+    #[error(transparent)]
+    Std(#[from] StdError),
```

Comment:
I'd prefer if you incorporated error-stack from the start, so we don't have to go back and redo the whole error reporting again. In that case this is not needed

Reply:
I thought this shouldn't be needed for the specific ContractError?